Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", ] as String[]
+def paths = []
+
+// using mesos node because it's a lightweight alpine docker image instead of full VM
+node('mesos') {
+  stage("Verify author") {
+    user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#tools-notify')
+  }
+}
+
+node('mesos-ubuntu') {
+  def shcmd = { String command ->
+      output = sh(script: command, returnStdout: true).trim()
+      println(output)
+      return output
+  }
+
+  checkout scm
+  stage("Get changeset") {
+    shcmd("git config remote.origin.fetch '+refs/heads/*:refs/remotes/origin/*'")
+    shcmd('git fetch --all')
+    branch = shcmd('git ls-remote --heads origin | grep $(git rev-parse HEAD) | cut -d / -f 3')
+    shcmd('git checkout master')
+    shcmd("git checkout ${branch}")
+    shcmd('git config --global user.name "mesosphere_jenkins"')
+    shcmd('git config --global user.email "mesosphere_jenkins"')
+    shcmd('git rebase master')
+
+    diffOutput = shcmd('git diff --name-only master')
+    changedFiles = diffOutput.split('\n')
+    for(file in changedFiles) {
+      println("changed file ${file}")
+      if (file.contains('install_dcos_prerequisites.sh') || file.contains('packer.json')
```

Comment:
These values can be put in a list and then checked with a `for` loop:

```
def fileNames = ["install", "packer", "build_and_test"]
for (f in fileNames) {
  if (file.contains(f) { ... }
}
```

Reply:
Will remove build_and_test so at a count of 2 checks only I will leave it in the if